app/serializers/artist_serializer.rb

```diff
@@ -1,3 +1,5 @@
 class ArtistSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :spotify_uri
+  embed :ids
+  has_many :songs
```

Comment:
My preference is to leave whitespace separating conceptually different pieces of information: blank line b/w validations & associations in a model, for example.  Here, would have blank lines b/f and after `embed :ids`